test/controllers/update/no.change.registrable.beneficial.owner.controller.spec.ts

```diff
@@ -131,7 +131,7 @@ describe("No change registrable beneficial owner", () => {
         .send({ [RegistrableBeneficialOwnerKey]: "1" });
 
       expect(resp.status).toEqual(302);
-      expect(resp.header.location).toEqual(UPDATE_REVIEW_STATEMENT_URL);
+      expect(resp.header.location).toEqual(UPDATE_STATEMENT_VALIDATION_ERRORS_URL);
```

Comment:
the test description needs updating for these 2 tests, it still says should redirect to UPDATE_REVIEW_STATEMENT_URL

Reply:
Changed description